src/Http/ServerRequest.php

```diff
@@ -2344,6 +2344,23 @@ public function getRequestTarget()
         return $target;
     }
 
+    /**
+     * Get the path of current request.
```

Comment:
Should we mention that this is missing the leading and trailing slashes?

Reply:
Shouldn't it include the leading slash at least?

Reply:
@inoas This is meant to be a replacement for `Request::$url` which does not include leading slash either.

That said I personally have almost always used that property after adding leading slash, so many in 4.0 we can modify the method to include leading slash.

Reply:
We could have the leading slash now. Adding it later is a subtle change that is likely going to cause problems. Doing it now will make the minor difference part of the upgrade process.

Reply:
@markstory I agree. Done so.